strawberry_django/mutations/fields.py

```diff
@@ -257,8 +259,9 @@ def create(self, data: dict[str, Any], *, info: Info):
 
 def get_pk(
     data: dict[str, Any],
+    key_attr: str,
```

Comment:
use pk (as explained below) and force this to be passed as a keyword argument

```suggestion
    *,
    key_attr: str | None = "pk",
```